From 236cce82ab25be73fb2a1cbd2f5bd71bff467e14 Mon Sep 17 00:00:00 2001
From: Jianzheng Zhou <jianzheng.zhou@freescale.com>
Date: Fri, 26 Jul 2013 15:32:06 +0800
Subject: [PATCH 10/14] ENGR00272735 wifi: add default compile object in rule

If no board wlan device is defined, it should also has a default
wifi.c be compiled.

Signed-off-by: Jianzheng Zhou <jianzheng.zhou@freescale.com>
---
 wifi/Android.mk | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/wifi/Android.mk b/wifi/Android.mk
index 44a191e..0e95753 100644
--- a/wifi/Android.mk
+++ b/wifi/Android.mk
@@ -55,6 +55,9 @@ ifeq ($(BOARD_WLAN_DEVICE),UNITE)
 else ifeq ($(BOARD_WLAN_VENDOR), INTEL)
   LOCAL_SRC_FILES += wifi/wifi_intel.c
   LOCAL_C_INCLUDES += $(LOCAL_PATH)/../../external/wpa_supplicant_8/src/common
+else
+  LOCAL_SRC_FILES += wifi/wifi.c
+  LOCAL_C_INCLUDES += $(LOCAL_PATH)/../../external/wpa_supplicant_8/src/common
 endif
 
 LOCAL_SHARED_LIBRARIES += libnetutils
-- 
1.8.0

